/*
#include <iostream> // Exceptions @ http://www.cplusplus.com/doc/tutorial/exceptions/
using namespace std;

int main () {
	try {
		throw 20;
	}
	catch (int e) {
		cout << "An exception occurred. Exception Nr. " << e << '\n';
	}
	return 0;
}
*/
